cloud_formation/configs/api.py

```diff
@@ -441,8 +441,25 @@ def update(session, domain):
     return success
 
 def delete(session, domain):
-    names = AWSNames(domain)
-    aws.route53_delete_records(session, domain, names.endpoint)
-    aws.sqs_delete_all(session, domain)
-    aws.policy_delete_all(session, domain, '/ingest/')
-    CloudFormationConfiguration('api', domain).delete(session)
+    yes = {'yes','y', 'ye', ''}
+    no = {'no','n'}
+
+    #Check the action is desired.
+    sys.stdout.write("Are you sure you want to proceed? All data within current boss will be lost.[y/n]")
+    choice = input().lower()
+    if choice in no:
+        print("Action cancelled.")
+    elif choice in yes:
+        names = AWSNames(domain)
+        aws.route53_delete_records(session, domain, names.endpoint)
+        aws.sqs_delete_all(session, domain)
+        aws.policy_delete_all(session, domain, '/ingest/')
+        CloudFormationConfiguration('api', domain).delete(session)
+    else:
+        print("Please respond with 'yes' or 'no'")
+
+    # names = AWSNames(domain)
```

Comment:
Why leave the commented out version?

Reply:
Didn't want to disregard it, but I'll just go ahead and delete it.